Mark basic_test.py rm_missing_with_case_clashing_ondisk_item as XFAIL.

From: David Rothenberger <daveroth@acm.org>

It depends on Windows code that can get the canonical case for an on
disk item, but that has not been implemented for Cygwin.
---
 subversion/tests/cmdline/basic_tests.py |    1 +
 1 file changed, 1 insertion(+)

diff --git a/subversion/tests/cmdline/basic_tests.py b/subversion/tests/cmdline/basic_tests.py
index 4b0b8d5..70b7e43 100755
--- a/subversion/tests/cmdline/basic_tests.py
+++ b/subversion/tests/cmdline/basic_tests.py
@@ -2955,6 +2955,7 @@ def quiet_commits(sbox):
 # Regression test for issue #4023: on Windows, 'svn rm' incorrectly deletes
 # on-disk file if it is case-clashing with intended (non-on-disk) target.
 @Issue(4023)
+@XFail()
 def rm_missing_with_case_clashing_ondisk_item(sbox):
   """rm missing item with case-clashing ondisk item"""
 
